UtilsDynamicSimulations/OpenSimAD/README.md

```diff
@@ -0,0 +1,83 @@
+- We generate [muscle-driven simulations](https://github.com/stanfordnmbl/opencap-processing/blob/main/UtilsDynamicSimulations/OpenSimAD/mainOpenSimAD.py#L999) that track joint kinematics. The general idea is to solve for the model controls that will drive the musculoskeletal model to closely track the measured kinematics while satisfying the dynamic equations describing muscle and skeletal dynamics and minimizing muscle effort. We use direct collocation methods to solve this problem, and leverage AD through [CasADi](https://web.casadi.org/).
+4. **Process results**
+- From the simulations, we can extract dynamic variables like muscle forces, joint moments, ground reaction forces, or joint contact forces.
+
+### Overview outputs
+- If your problem converges, you should get a few files under OpenSimData/Dynamics:
+  - forces_<trial_name>.mot
+    - Muscle forces and non muscle-driven joint torques (eg, reserve actuators).
+  - GRF_resultant_<trial_name>.mot
+    - Resultant ground reaction forces and moments.
+  - GRF_<trial_name>.mot
+    - Ground reaction forces and moments (per contact sphere).
+  - kinematics_activations_<trial_name>.mot
+    - Joint kinematics and muscle activations.
+  - kinetics_<trial_name>.mot
+    - Net joint moments.
+  - optimaltrajectories.npy
+    - Dictionary with compiled results:
+      - time: discretized time vector.
+      - coordinate_values_toTrack: reference coordinate values estimated from videos.
+      - coordinate_values: coordinate values resulting from the dynamic simulation.
+      - coordinate_speeds_toTrack: reference coordinate speeds estimated from videos.
+      - coordinate_speeds: coordinate speeds resulting from the dynamic simulation.
+      - coordinate_accelerations_toTrack: reference coordinate accelerations estimated from videos.
+      - coordinate_accelerations: coordinate accelerations resulting from the dynamic simulation.
+      - torques: joint torques/moments from the dynamic simulation.
+      - torques_BWht: joint torques/moments normalized by body weight times height from the dynamic simulation.
+      - GRF: resultant ground reaction forces from the dynamic simulation.
+      - GRF_BW: resultant ground reaction forces normalized by body weight from the dynamic simulation.
+      - GRM: resultant ground reaction moments from the dynamic simulation, expressed with respect to global reference frame.
+      - GRM_BWht: resultant ground reaction moments normalized by body weight times height from the dynamic simulation.
+      - muscle_activations: muscle activations from the dynamic simulation.
+      - passive_muscle_torques: passive torque contribution of the muscles from the dynamic simulation.
+      - active_muscle_torques: active torque contribution of the muscles from the dynamic simulation.
+      - passive_limit_torques: torque contributions from limit torques from the dynamic simulation.
+      - KAM: knee adduction moments from the dynamic simulation.

```

Comment:
this isn't computed for all activities is it? If not, we may want to say (optionally)

Reply:
I have been going back and forth with defaulting to True or False [here](https://github.com/stanfordnmbl/opencap-processing/blob/main/UtilsDynamicSimulations/OpenSimAD/mainOpenSimAD.py#L38). What do you think?

Reply:
I might go true.  You make it clear that everything needs validating. It will avoid a lot of "where's the KAM" questions.